server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -214,6 +214,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ESQL_REMOVE_AGGREGATE_TYPE = def(9_045_0_00);
     public static final TransportVersion ADD_PROJECT_ID_TO_DSL_ERROR_INFO = def(9_046_0_00);
     public static final TransportVersion SEMANTIC_TEXT_CHUNKING_CONFIG = def(9_047_00_0);
+    public static final TransportVersion ML_INFERENCE_ELASTIC_RERANK = def(9_048_0_00);
```

Comment:
Looks like the PR is only targeting 9.1 did we also want to support 8.19? If so we'll need to add another transport version and do the backport dance.

Reply:
It needs to go into `9.1` & `8.19`